static/js/components/tiles/histogram_tile.tsx

```diff
@@ -18,7 +18,7 @@
  * Component for rendering a histogram type tile.
  */
 
-import _ from "lodash";
+import _, { property } from "lodash";
```

Comment:
i don't think this `property` is being used?

Reply:
Thanks for the catch, that was auto-added by my IDE by accident. Removed.